src/ui/components/filters/filteroptionscomponent.js

```diff
@@ -658,7 +664,6 @@ export default class FilterOptionsComponent extends Component {
         remove: () => this._clearSingleOption(o)
       }));
 
-    this.core.persistentStorage.set(this.name, this.config.options.filter(o => o.selected).map(o => o.label));
```

Comment:
FilterOptions is intended to also work as a standalone component, so it still needs to store stuff in persistent storage in that case as well, (though I don't think many people (if any) use it standalone, it's still something we support)

Reply:
Good catch. I'll take a look